package.json

```diff
@@ -3,13 +3,17 @@
   "version": "7.2.0-rc-cubalibre.4",
   "main": "./index.js",
   "dependencies": {
+    "fs-plus": "3.0.2",
     "json-stringify-deterministic": "1.0.1",
     "lodash.merge": "4.6.0",
-    "through2": "2.0.3",
     "mssql": "4.1.0",
+    "through2": "2.0.3",
     "uuid": "3.2.1",
     "xml2js": "0.4.19"
   },
+  "peerDependencies": {
+    "tedious": "^2.0.0"
```

Comment:
Isn't this already a peer dependency of mssql?

Reply:
It is a [dependency](https://github.com/tediousjs/node-mssql/blob/master/package.json#L34) of mssql. And I saw it is directly required [here](https://github.com/softwaregroup-bg/ut-port-sql/blob/minor/rc-cubalibre/index.js#L25) so I had to add it to the package.json. If we end up with 2 different versions of tedious that patch won't work.